### PR TITLE
REG-383 No visible control to close the help window in the markdown editor

### PR DESCRIPTION
**Pull request by OlexMir**
No visible control to close the help window in the markdown editor

### CODE REVIEW

**Comment by tbookless** [comment]
@tveronezi and @OlexMir is this a change to our code or the component's code? Do we need to do a PR on the MD component project?

**Comment by tveronezi** [comment]
Thanks for the update @OlexMir 

One last minor thing and we are good to go. Can you change the position just a tinny bit, so the button is in equal distance from the top and right borders? It will also avoid having the icon on top of the content.

Currently is looks like this...
screen shot 2016-11-18 at 11 35 45 am

With the change, it will look like this...
screen shot 2016-11-18 at 11 32 57 am
screen shot 2016-11-18 at 11 33 04 am

File: tribestream-api-registry-webapp/src/main/static/assets/styles/_component_markdown.sass [hunk 1]
```diff
@@ -49,11 +49,15 @@ div.markdown-help-content
...
-    padding: 40px
+    padding: 40px 50px 40px 50px
     overflow: auto
     background-color: white
     @include code-style
-
+    >i.fa.fa-times-circle
+      position: fixed
+      top: 55px
+      right: 65px
```
Comment (tveronezi) [inline tribestream-api-registry-webapp/src/main/static/assets/styles/_component_markdown.sass]: Does not look like its aligned. Change it to 55px to get proper alignement. 

screen shot 2016-11-21 at 9 17 55 am

screen shot 2016-11-21 at 9 18 44 am

**Comment by OlexMir** [comment]
Thanks for your reply. I want explain why i choose 65px, and not 55px. CSS does not understand if there scrollbar or not. So if i have scrollbar and 55px indent, I will have not pretty situation with our button (look on screenshot). Otherwise, If i set 65px i will have good position and display on both ways.

65px:
65px

55px:
55px

**Comment by tbookless** [comment]
fixes #222

**Pull request merged**